randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -328,8 +328,10 @@ Extra - asset_id: 8
   > Door to Forgotten Highway Kago Room
       All of the following:
           Morph Ball
-          # With Wall Jumps: https://youtu.be/Z_vt91bLKeA
-          Hi-Jump Boots or Space Jump or Wall Jumps (Beginner)
+          Any of the following:
+              # With Wall Jumps: https://youtu.be/Z_vt91bLKeA
+              Hi-Jump Boots or Space Jump or Wall Jumps (Beginner)
+              Gravity Suit and Speed Booster and Shinespark (Beginner) and Short Charge (Intermediate)
```

Comment:
Does crab maze *really* need an intermediate short charge?

Reply:
The charge puts you in the next room. 1 tap keeps you in the same room. I'd have to add "and not door rando" or something

Reply:
Huh, maybe im too used to am2r where its significantly shorter
In that case, just put a `short cahrge intermediate OR (no door lock rando)`

Reply:
done